### PR TITLE
Add xdsmjs

### PR DESCRIPTION
**Pull request by AntoineD**
@conda-forge/help-python

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AntoineD** [comment]
@conda-forge/help-python ready for review

File: recipes/xdsmjs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (mariusvniekerk) [inline recipes/xdsmjs/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/xdsmjs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
```
Comment (mariusvniekerk) [inline recipes/xdsmjs/meta.yaml:23]:
```suggestion
    - python >=3.6
```

**Review by mariusvniekerk — CHANGES_REQUESTED** [review]
According to https://github.com/OneraHub/XDSMjs/blob/master/py_xdsmjs/setup.py there is a minimum of python 3 fpr this library

File: recipes/xdsmjs/meta.yaml (hunk 1 above)
Comment (AntoineD) [reply recipes/xdsmjs/meta.yaml:21]: Yes but it works fine for python 2.7, this package is just a [dumb wrapper](https://github.com/OneraHub/XDSMjs/blob/master/py_xdsmjs/xdsmjs/__init__.py) with only an `__init__.py`.
I will add a lower bound though.

**Comment by AntoineD** [comment]
@conda-forge/help-python ready for review

**Comment by AntoineD** [comment]
@conda-forge/help-python Is there something still blocking? Could anybody please have a look?

**Pull request merged**